.github/workflows/main.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: Build
+      id: build
+      uses: convox/actions/build@multipart
+      env:
+        CONVOX_PASSWORD: ${{ secrets.CONVOX_DEPLOY_KEY}}
+      with:
+        CONVOX_HOST: console.convox.com
```

Comment:
would be nice if these were `host`, `rack`, and `app` parameter names

the action should default `host` to `console.convox.com` if not specified

Reply:
just so I understand do you want me to drop the "CONVOX_" prefix and it just be "HOST" for example?

Reply:
Yes, and lowercase (the names i put in code blocks in the comment)